### PR TITLE
Gerald/dns zones

### PR DESCRIPTION
**Pull request by futuregerald**
Adding new dns_zone endpoints

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -1501,6 +1500,143 @@ paths:
...
+  /dns_zones/{zone_id}/transfer:
+    parameters:
+      - name: zone_id
+        type: string
+        in: path
+        required: true
+      - name: account_id
+        type: string
+        in: query
+        description: 'the account you want to transfer the dns zone to'
```
Comment (kitop) [inline swagger.yml]: This is the `account_id` of the Dns Zone.

The endpoint requires 2 more params:
- `transfer_account_id` - the account you want to transfer the dns zone to
- `transfer_user_id` - the user you want to transfer the dns zone to (can be the same as the user calling the endpoint)

**Comment by oscartbeaumont** [comment]
Is the change going to be implemented and this PR merged? I am working on a project and having a first-party API client for Netlify Managed DNS would be much nicer than using my custom implementation.

**Comment by mraerino** [comment]
@futuregerald do wou want to pick this up again? otherwise i'd like to clase as stale

**Comment by bertvansteen** [comment]
I merged master and implemented the suggested change in a forked repo, feel free to pull these changes into the PR branch: https://github.com/cloudtactic/open-api/tree/gerald/dns_zones

Or should I create a new PR to open-api/master?

**Comment by kitop** [comment]
> I merged master and implemented the suggested change in a forked repo, feel free to pull these changes into the PR branch: https://github.com/cloudtactic/open-api/tree/gerald/dns_zones
> Or should I create a new PR to open-api/master?

@bertvansteen Thanks. Feel free to open a PR!

**Comment by oscartbeaumont** [comment]
I have already done that in PR #207 so it shouldn't be necessary.

**Pull request closed without merging**